sql/src/test/java/org/opensearch/sql/sql/parser/AstExpressionBuilderTest.java

```diff
@@ -451,6 +451,23 @@ public void filteredDistinctCount() {
     );
   }
 
+  //TODO: Verify what this test does
```

Comment:
TODO?

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/165/commits/d04916bfbe558347bec18a9b61b3f13697ce270d